man/man8/swtpm.pod

```diff
@@ -150,14 +150,18 @@ The following options are support by all interfaces:
 
 =over 4
 
-=item B<--tpmstate dir=E<lt>dirE<gt>[,mode=E<lt>0...E<gt>]>
+=item B<--tpmstate dir=E<lt>dirE<gt>[,mode=E<lt>0...E<gt>]|backend_uri=E<lt>uriE<gt>>
```

Comment:
Hopefully last change. Please change this to `backend-uri` to be consistent with other option parameters.

Reply:
The help screens in swtpm.c, swtpm_chardev.c, and cuse_tpm also need to be extended.

Reply:
Sure, I'll update them.